Fix error that prevents removing the compressed file when decompressing:

Failed to unlink (null): Bad address
Fatal error - exiting
--- runzip.c.orig	2010-12-19 12:12:30.000000000 -0600
+++ runzip.c	2013-02-21 10:28:58.000000000 -0600
@@ -181,7 +181,6 @@
 {
 	off_t total = 0;
 
-	control->infile = 0;
 	while (total < expected_size) {
 		total += runzip_chunk(fd_in, fd_out, fd_hist);
 	}
